_includes/markdown/Tools.md

```diff
@@ -23,14 +23,16 @@ is `docker.test`, you can access PHPMemcachedAdmin at `docker.test/__memcacheadm
 
 ## Xdebug
 
-Xdebug is included in the php images and is nearly ready to go out of the box. Make sure your IDE is listening for
+Xdebug is included in the php images but must be manually enabled.  To enable Xdebug, set the enviornment variable
+`ENABLE_XDEBUG` to `'true'` in the `docker-compose.yml` file in the root of the project. Make sure your IDE is listening for
 PHP debug connections and set up a path mapping to your local environment's `wordpress/` directory to `/var/www/html/`
 in the container.
 
 ### Visual Studio Code
-1. Install the [PHP Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
-2. In your project, go to the Debug view, click "Add Configuration..." and choose PHP environment. A new launch configuration will be created for you.
-3. Set the `pathMappings` parameter to your local `wordpress` directory. Example:
+1. Ensure Xdebug is enabled for the environment using the `ENABLE_XDEBUG` environment variable.
```

Comment:
In my case, it took me some minutes to find out that I need to enable xdebug in `docker-composer.yml` file, so I think we should be more explicit here about that.

And I don't know if it happens to others, but for me, I have to stop the environment and start it again to have xdebug enabled, restarting didn't help.